tests/test_compat.py

```diff
@@ -1,24 +1,87 @@
-# SPDX-FileCopyrightText: 2021 Jens Erdmann
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: 2022 Henrik Sandklef
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-import json
-import pytest
+import unittest
```

Comment:
why the fallback on legacy `unitest` - I'd prefer sticking with way mightier `pytest`

Reply:
Suggest switching to that for all files in separate branch.